cEP-0009.md

```diff
@@ -0,0 +1,88 @@
+cobot Enhancement and porting
+=============================
+
+|Metadata|                                       |
+|--------|---------------------------------------|
+|cEP     |0009                                   |
+|Version |1.0                                    |
+|Title   |cobot Enhancement and porting          |
+|Authors |Meet Mangukiya <meetmangukiya@coala.io>|
+|Status  |Proposed                               |
+|Type    |Process                                |
+
+Abstract
+--------
+
+This cEP describes the details of the process of porting
+[cobot](https://gitlab.com/coala/cobot) from hubot(coffeescript framework) to
+[errbot](http://errbot.io)(python framework) and about the new features that are
+to be added to cobot as a part of the 
+[GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).
+
+## New Features
+
+### Search API and User docs
+
+Construct the url from the search string and return it as a message.
+
+```
+> cobot search api|user this is a search string
+Have a look at this : http://api.coala.io/en/latest/search.html?q=this+is+a+search+string
+```
+
+### List Bears
+
+Query webservices.coala.io according to the filters that are applied/acquired
+from the trigger command.
+
+```
+> cobot ls bears [lang]*
+### lang1
+1. ...
+2. ...
+3. ...
+### lang2
+1. ...
+2. ...
+3. ...
```

Comment:
Can you describe how the output would look like and a brief overview of how the command will work on the backend. Like errbot will make a request to which api endpoint of webservices and so on